model/task/db.go

```diff
@@ -381,6 +382,14 @@ func ByDispatchedWithIdsVersionAndStatus(taskIds []string, versionId string, sta
 	})
 }
 
+func ByExecutionTask(taskId string) db.Q {
+	return db.Query(bson.M{
+		ExecutionTasksKey: bson.M{
+			"$in": []string{taskId},
```

Comment:
shouldn't this be an equality query? there's only one value?

Reply:
That field is an array of values, so equality won't work. Let me know if there's a cleaner way to query for a string array containing a specific string.